notebooks/basics/multiple-systems.ipynb

```diff
@@ -184,26 +184,26 @@
     "    \\leftarrow \\text{probability associated with state 11}\n",
     "  \\end{array}\n",
     "  \\label{eq:correlatedbits} \\tag{1}\n",
-    "$$"
+    "$$\n",
+    "\n"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "bffda1a9",
    "metadata": {},
    "source": [
-    "#### Independence for two systems\n",
+    "#### Independence of two systems\n",
     "\n",
     "A special type of probabilistic state of two systems is one in which the systems are *independent*.\n",
-    "Intuitively speaking, two systems are independent if learning the classical state of either one of the systems has no effect on the probability associated with each state of the other.\n",
-    "That is, learning what classical state one of the systems is in provides no information at all about the classical state of the other.\n",
+    "Intuitively speaking, two systems are independent if the state of one system has no bearing on the state of the other. Given access to knowledge of the state of one system, perhaps via measurement, the probabilities I assign to the states of the other system do not change. Put another way, the information required to learn about each system is completely contained within that system, not to be found elsewhere. \n",
```

Comment:
Intuitively speaking, two systems are independent if the state of one system has no bearing on the state of the other. Given two independent systems, gaining access to knowledge of the state of one system does not change the probabilities we assign to the states of the other.

[I don't like the sentence about information not being found elsewhere, because there could be additional systems that do contain that information.]